grpc/testing/compiler_test.proto

```diff
@@ -0,0 +1,71 @@
+// Copyright 2016 gRPC authors.
```

Comment:
This file may have been omitted on purpose.

I wonder if it wouldn't ultimately be best for grpc/grpc to move this to another proto package and test independently.  Unless the tests for the compiler correctness also are testing other proto messages/etc in this proto package.

Reply:
Excluded.